imports/plugins/core/accounts/client/templates/members/member.js

```diff
@@ -141,6 +143,15 @@ Template.memberSettings.helpers({
       component: ManageGroups,
       ...Template.currentData()
     };
+  },
+
+  ReactionAvatar() {
+    const user = Accounts.findOne(Meteor.userId());
```

Comment:
All of this data fetching logic should be in a container for the `ReactionAvatar` component, not the Blaze template.  That makes overriding the React component that much more complicated.